### DISCUSSION TITLE
How to fill a Matrix in a resource with records from related table

### DISCUSSION THREAD

**Discussion by rabol** [orchidsoftware/platform#1514] [Q&A]
Imagine this:

A product order, two models, ProductOrder, ProductLine. One ProductOrder can have n ProductLine(s):

Create a resource for ProductOrder
add the the needed fields in the fields() method and add a Matrix for entering lines

That works fine

but if I then want to edit the 'ProductOrder', how do I then get the Matrix filled withe the ProductLine(s) that is related to the  ProductOrder?

**Comment by sylzys** [comment]
Same question here, any idea @tabuna ? thanks :)
